.github/workflows/ci.yml

```diff
@@ -69,7 +69,7 @@ jobs:
     if: needs.detect-changes.outputs.java-code-changes == 'true'
     needs: [detect-changes]
     name: Java checks
-    timeout-minutes: 30
+    timeout-minutes: 15
```

Comment:
I think the rules are that the timeout cannot be longer than 10 mins. See [this](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria) document for more details

Reply:
This is reverting changes that were in the monorepo already. That particular job takes approx. 13 minutes. This comment has a bit more context about this particular timeout: https://github.com/camunda/camunda/pull/19381#issuecomment-2258092804